From 651bd680f26d23a245013459ef40353073d71c5e Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:31 -0400
Subject: [redhat] kabi: Adding symbol bio_alloc_bioset (fs/bio.c)

Message-id: <b848258301969d4b62e5930d56df887604e60b70.1410510773.git.poros@redhat.com>
Patchwork-id: 95923
O-Subject: [PATCH RHEL7.1 031/233] kabi: Adding symbol bio_alloc_bioset (fs/bio.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Not used directly. bio_alloc used by PowerPath

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/bio_alloc_bioset
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/bio_alloc_bioset
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/bio_alloc_bioset

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/bio_alloc_bioset b/redhat/kabi/kabi-module/kabi_ppc64/bio_alloc_bioset
new file mode 100644
index 0000000..70c3ec8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/bio_alloc_bioset
@@ -0,0 +1,2 @@
+#1-
+0xe0a41dcf bio_alloc_bioset vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/bio_alloc_bioset b/redhat/kabi/kabi-module/kabi_s390x/bio_alloc_bioset
new file mode 100644
index 0000000..296fde4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/bio_alloc_bioset
@@ -0,0 +1,2 @@
+#1-
+0xb17b25f4 bio_alloc_bioset vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/bio_alloc_bioset b/redhat/kabi/kabi-module/kabi_x86_64/bio_alloc_bioset
new file mode 100644
index 0000000..135368b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/bio_alloc_bioset
@@ -0,0 +1,2 @@
+#1-
+0xeb862489 bio_alloc_bioset vmlinux EXPORT_SYMBOL
-- 
1.7.1